components/script/script_thread.rs

```diff
@@ -2857,6 +2864,7 @@ impl ScriptThread {
             self.microtask_queue.clone(),
             self.webrender_document,
             self.webrender_api_sender.clone(),
+            self.layout_thread_is_busy.clone(),
```

Comment:
Each Window should get its own busy boolean, since there is a 1:1 window:layout thread ratio.

Reply:
Does this mean there is a separate layout thread for every window?

Reply:
Each window has its own layout thread, yes.